grid-cli/cmd/deploy_vm.go

```diff
@@ -152,40 +128,38 @@ var deployVMCmd = &cobra.Command{
 			log.Fatal().Err(err).Send()
 		}
 
-		if node == 0 {
-			filter, ssd, rootfss := filters.BuildVMFilter(vm, diskMount, volumeMount, farm)
-			nodes, err := deployer.FilterNodes(
-				cmd.Context(),
-				t,
-				filter,
-				ssd,
-				nil,
-				rootfss,
-			)
-			if err != nil {
-				log.Fatal().Err(err).Send()
+		// if no public ips or yggdrasil then we should go for the light deployment
```

Comment:
We should go for the light one first, but if there're no enough nodes found to deploy, we should deploy the normal Vm